src/Sulu/Bundle/WebsiteBundle/Navigation/NavigationMapper.php

```diff
@@ -239,4 +229,17 @@ public function inNavigation(StructureInterface $content, $context = null)
         // do not show
         return false;
     }
+
+    private function normalizeResult(array $result)
+    {
+        foreach ($result as &$item) {
```

Comment:
Since you are returning the new array and reassigning it above anyway, wouldn't it be less error-prone to return a new copy here, instead of altering the existing references?